anyatibrian/IRent-backend
# 168691064 implement user login

#### Description
Users should be able to login with an email and password
#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test
- Hit the endpoint /api/v1/users/login with a username and password
- If credentials are valid an access token will be generated
Give instructions on how reviewers can verify your work

#### Any background context you want to add
- Authentications enables verified users to be able to login and access the system
#### Story-ID

[Starts #168691064]
